scripts/welcome.js

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
```

Comment:
Our friend the welcome.js file does not need to return :)

I'm pushing a commit to purge it and using the opportunity to see if `auto` reacts to PR labels now being set.